ui/cypress/integration/tests.js

```diff
@@ -37,6 +37,8 @@ describe("Basic tests", () => {
     cy.get("input[name='queries-mode']").click();
     cy.contains("SELECT *");
 
+    cy.reload();
```

Comment:
nit: possibly a comment here, if this is testing something around persistence.

Possibly also, should it check that the prior cohort appears in the list still?